Automattic/vip-go-mu-plugins
# Updating VaultPress to 2.2.0

## Description

This PR fetches the latest version of the VaultPress plugin, 2.2.0.

## Changelog Description

### Plugin Updated: VaultPress 2.2.0

We upgraded VaultPress 2.1.4 to VaultPress 2.2.0

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Make sure Jetpack and VaultPress are enabled.
3. Check that the VaultPress UI works on wp-admin.
4. On a sandbox, check that backups still run.
